packages/block-library/src/gallery/block.json

```diff
@@ -72,14 +79,33 @@
 			"type": "boolean",
 			"default": true
 		},
+		"linkTarget": {
+			"type": "string"
+		},
 		"linkTo": {
 			"type": "string"
 		},
 		"sizeSlug": {
 			"type": "string",
 			"default": "large"
+		},
+		"allowResize": {
+			"type": "boolean",
+			"default": false
+		},
+		"isGrouped": {
+			"type": "boolean",
+			"default": true
+		},
```

Comment:
Do we really need to introduce this context at all, isn't this the same thing as checking that the image is a children of a gallery block using the data module? (We already do something similar for "post blocks" inside Query blocks, the behavior of post blocks can change slightly if they are children of query blocks or not)

Reply:
@talldan, @youknowriad - I agree that it would be good to explore alternatives to this. I wonder if coupling it tightly to a specific parent blocks via the data module is a better way forward though? ie. by passing in the new context values any new block can group images without having to modify the image block, but using the data module the image block would need updating to add new parent blocks - I may be misunderstanding what you are suggesting though.

Either way, since we are not serializing these values, so no deprecations needed for changes, would you both be happy if we handle this as a follow up PR soon after this one is merged? I think it will be easier to explore options for this in isolation from all the other changes in this PR. I have added [an issue for it](https://github.com/WordPress/gutenberg/issues/33531). Let me know if you think it needs following up before merging.

Reply:
It  turned out that `isGrouped` was not actually needed anymore since we moved away from using an ordered list for display, and since the [convert to cover block image toolbar option was made more specific ](https://github.com/WordPress/gutenberg/pull/33095)- so removed here. So just the allowResize context left.